app/assets/stylesheets/white_theme/user.scss

```diff
@@ -74,6 +71,73 @@ main {
         li:last-child {
           margin-right: 36px;
         }
+        @media #{$one-column} {
+          justify-content: space-between;
+          margin-bottom: -$baseline * 1.5;
+          margin-top: $baseline * 1.5;
+          margin-right: 0;
+          li:last-child {
+            margin-right: 0px;
+          }
+          li {
+            display: block!important;
+            width: 45%;
+            margin: 0;
+            margin-bottom: $baseline * 1.5;
+            img {
+              width: 100%;
+            }
+            svg {
+              width: 100%;
+            }
+          }
+        }
+
+        @media #{$mobile} {
+          width: 100%;
+          margin-bottom: -$baseline * .75;
+          justify-content: space-around;
+          li {
+            display: block!important;
+            width: 45%;
+            margin: 0;
+            margin-bottom: $baseline * .75;
+          }
+        }
+
+        @media only screen and (max-width: 1038px) {
+          li {
+            &:nth-child(1), &:nth-child(2), &:nth-child(3) {
+              display: block;
+            }
+            &:nth-child(3) {
+              margin-right: 0;
+            }
+            &:nth-child(4) {
+              display: none;
+            }
+          }
+        }
+        @media only screen and (max-width: 798px) {
+          li {
+            &:nth-child(1), &:nth-child(2) {
+              display: block;
+            }
+            &:nth-child(3), &:nth-child(4),  {
+              display: none;
+            }
+          }
+        }
+      }
+      .below_box {
+        margin-top: -36px;
+        margin-bottom: 26px;
+        @media #{$mobile} {
+          margin-top: 0;
+        }
+        &.sparse {
+          text-align: left;
+        }
```

Comment:
@ofsound Just FYI I talked with smoofles and he recommended putting the button on the left side when there's only 2/3 playlists. So I added some logic to do that, as well as this classname, and increased the bottom margin a bit...